Scheduler/FirstComeFirstServe.c

```diff
@@ -0,0 +1,103 @@
+/**
+ * @file FirstComeFirstServe.c
+ */
+
+#include "FirstComeFirstServe.h"
+#include "ScheduleFile.h"
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdbool.h>
+
+void RunFCFSProcess (ScheduleData* inputData){
+	bool debug = false;		
+	
+	if(debug) printf("Starting FCFS\n");
+
+	FILE *out = fopen("processes.out", "w");
+	if(out == NULL) {
+		printf("Error opening output file\n");
+		return;
+	}
+	if(debug) printf("Opened processes.out\n");
+
+	int numProc = VectorCount(inputData->processes);
+
+	if(debug) printf("%d processes\nUsing First-Come First-Serve\n\n", numProc);
+	fprintf(out, "%d processes\nUsing First-Come First-Serve\n\n", numProc);
```

Comment:
The final small thing. Just change "First-Come First-Serve" back to "First Come First Served".